ModuleWork1/ModuleWork1/Program.cs

```diff
@@ -0,0 +1,292 @@
+﻿string? command;
+string? task;
+string status;
+var dateFormat = "dd.MM.yyyy HH:mm";
+const string validChars = "-_ .*";
+var col = 3;
+var items = new string[1, col];
+
+InputCommand();
+
+void InputCommand() 
+{
+    while (true)
+    {
+        Console.WriteLine("Input one of this command: add-item, remove-item, mark-as, show, exit.");
+        command = Console.ReadLine();
+
+        
+        string[] commandsArr = { "add-item", "remove-item", "mark-as", "show", "exit" };
+
+        for (var i = 0; i < commandsArr.Length; i++)
+        {
+            if (command == commandsArr[i])
+            {
+                switch (commandsArr[i])
+                {
+                    case "add-item":
+                        Console.WriteLine("A");
+                        AddItem();
+                        break;
+                    case "remove-item":
+                        Console.WriteLine("R");
+                        RemoveItem();
+                        break;
+                    case "mark-as":
+                        Console.WriteLine("M");
+                        MarkItem();
+                        break;
+                    case "show":
+                        Console.WriteLine("S");
+                        Show();
+                        break;
+                    case "exit":
+                        return;
+                }
+            }
+        }
+    }
+}
+
+
+string[,] AddTask(string[,] array, string element)
+{
+    var rows = array.GetLength(0);
+    var cols = array.GetLength(1);
+
+    var newArr = new string[rows + 1, cols];
+
+    for (var i = 0; i < rows; i++)
+    {
+        for (var j = 0; j < cols; j++)
+        {
+            newArr[i, j] = array[i, j];
+        }
+    }
+
+    newArr[rows, 0] = element;
+
+    return newArr;
+}
+
+string[,] RemoveTask(string[,] array, int index)
+{
+    var rows = array.GetLength(0);
+    var cols = array.GetLength(1);
+
+    var newArray = new string[rows - 1, cols];
+
+    var newArrayIndex = 0;
+
+    for (var i = 0; i < rows; i++)
+    {
+        if (i != index)
+        {
+            for (var j = 0; j < cols; j++)
+            {
+                newArray[newArrayIndex, j] = array[i, j];
+            }
+            newArrayIndex++;
+        }
+    }
+
+    return newArray;
+}
+
+void AddItem()
+{
+    Console.WriteLine("Add task:");
+
+    task = Console.ReadLine();
+
+    if (!string.IsNullOrEmpty(task) &&
```

Comment:
This code is very similar to what you have in RemoveItem. Probably should be moved to the common function that does search.